cmake/Modules/UseDoxygen.cmake

```diff
@@ -134,7 +159,9 @@ if(DOXYGEN_FOUND AND DOXYFILE_IN_FOUND)
 
 	configure_file("${DOXYFILE_IN}" "${DOXYFILE}" @ONLY)
 
-	get_target_property(DOC_TARGET doc TYPE)
+    if(TARGET doc)
```

Comment:
Just a small thing - should this follow the rest of the cmake files and be indented with tabs?

Reply:
...Did I not leave my previous responses as replies to this comment? I _thought_ I did. Not helping, Github.